src/main/webapp/src/app/schedule-page/schedule-page.component.css

```diff
@@ -0,0 +1,38 @@
+/* Split the screen in half */
```

Comment:
Another way to do this is with CSS flexbox btw.

Reply:
I'm not a flexbox pro, so I just made a little demo to try it out: https://jsfiddle.net/aq742bcy/5/

Reply:
I just did! Thank you for the demo